.gitignore

```diff
@@ -4,6 +4,7 @@
 /.idea/libraries
 .DS_Store
 /build
+/captures
```

Comment:
This is a directory Android Studio creates when you try to dump memory from within AS.